### ISSUE TITLE
Warnings running ldd on shared objects extracted from pyinstaller

### ISSUE THREAD

**Issue by JonathonReinhart** [JonathonReinhart/staticx#121]
```
staticx dist/myBin dist_static/myBin
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_asyncio.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_bisect.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_blake2.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_bz2.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_codecs_cn.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_codecs_hk.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_codecs_iso2022.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_codecs_jp.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_codecs_kr.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_codecs_tw.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_contextvars.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_csv.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_ctypes.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_datetime.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_decimal.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_elementtree.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_hashlib.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_heapq.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_json.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_lzma.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_md5.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_multibytecodec.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_multiprocessing.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_opcode.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_pickle.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_posixshmem.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_posixsubprocess.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_queue.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_random.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_sha1.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_sha256.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_sha3.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_sha512.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_socket.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_ssl.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/_struct.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/array.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/binascii.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/grp.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/ld-linux-x86-64.so.2'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/libbz2.so.1.0'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/libcrypto.so.1.1'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/libexpat.so.1'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/libffi.so.6'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/liblzma.so.5'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/libmpdec.so.2'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/libncursesw.so.6'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/libpython3.8.so.1.0'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/libreadline.so.8'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/libssl.so.1.1'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/libudev.so.1'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/libusb-1.0.so.0'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/libz.so.1'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/math.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/mmap.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/pyexpat.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/readline.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/resource.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/select.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/termios.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/unicodedata.cpython-38-x86_64-linux-gnu.so'
ldd: warning: you do not have execution permission for `/tmp/staticx-pyi-fe35tk1i/zlib.cpython-38-x86_64-linux-gnu.so'
```

_Originally posted by @frankenstein91 in https://github.com/JonathonReinhart/staticx/issues/116#issuecomment-579765281_

**Comment by JonathonReinhart** [comment]
Fixed by #122.

**Comment by luis-lavaire** [comment]
Hi!

`pip3 install staticx; staticx ...` still emits those warnings for me, although the generated binaries do work. Is that an expected and normal behavior?
